cookbooks/cdo-varnish/libraries/http_cache.rb

```diff
@@ -139,7 +139,6 @@ def self.config(env)
               /amazon-future-engineer*
               /create-company-profile*
               /edit-company-profile*
-              /teacher-dashboard*
```

Comment:
@Hamms this is delayed and partial follow up to the conversation we had about potential ways to clean up the special casing in `http_cache`. What do you think?